### ISSUE TITLE
C-x C-e on (modalka-define-kbd "x" "M-x") prints garbage?

### ISSUE THREAD

**Issue by rdotdk** [mrkkrp/modalka#9]
```C-x C-e``` on ```(modalka-define-kbd "x" "M-x")``` prints garbage?

My config:
```
(require 'modalka)
(modalka-define-kbd "x" "M-x") 
```

Placing the cursor at the end of modalka-define-kbd and pressing ```C-x C-e``` prints the following output.
```

#[0 "ÁÀ!\211Â>\206^L^@Ã^A!?\205^S^@Ä^A!\207" [[134217848] key-binding (nil undefined) keymapp call-interactively] 3 "

(fn)" nil]

```
With the above config, if i switch to ``` modalka-mode ``` and press ``` x ``` it invokes  ``` M-x ``` but when i try to evaluate the line using  ``` C-x C-e ``` it gives this prints. Is this expected?

**Comment by mrkkrp** [comment]
Yes, this is textual representation of `modalka-mode-map`.

**Comment by phil-s** [comment]
Moreover, `C-h c` and `C-h k` give you garbage for the modalka bindings. That's a deal-breaker, to my mind.

**Comment by mrkkrp** [comment]
This is coded this way because we just translate key presses. To have helpful output from <kbd>C-h c</kbd> and <kbd>C-h k</kbd> you need to bind actual commands and this would be *real* deal-breaker. Because commands key-bindings invoke do change depending on many things. For example there may be a command in key map of some overlay. When point in on that overlay, some key press has one meaning and in other cases another. Modalka in its present state handles this flawlessly.

Of course I don't like this unhelpful output too, but I don't know how to do it better (and perhaps nobody knows). You just call help commands on *original* bindings, that's it.

This reminds me to close the issue because if some function returns a keymap and it happens to look not very pretty. It's a bug in any sense.

**Comment by mrkkrp** [comment]
@phil-s, I'll take a look how to improve output of <kbd>C-h k</kbd> on this weekend.

**Comment by phil-s** [comment]
If you wouldn't mind following up in this issue if you come up with a solution, that would be great (as I'll get a notification that way).

The following (using an uninterned symbol) might be a useful approach:

    (global-set-key "F" (defalias (make-symbol "my-fake-function") (lambda () (interactive) (insert "F")) "Fake function docstring"))

**Comment by mrkkrp** [comment]
@phil-s, I opened #10 to keep track of this problem.

**Comment by phil-s** [comment]
@rdotdk: FYI a side-effect of the fix for #10 is that it also cleans up the output you were seeing here.
